docs/tutorials/05_kv-scheduler.md

```diff
@@ -39,8 +39,8 @@ northbound proto models, the Orchestrator needs message names to be present in t
 message names are present in proto messages we must use the following special protobuf option (together with its
 import):
 ```proto
-import "github.com/gogo/protobuf/gogoproto/gogo.proto";
-option (gogoproto.messagename_all) = true;
+//import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+//option (gogoproto.messagename_all) = true;
```

Comment:
hehe, we must use those comments, so important :)
(not reviewing, just looking around)

Reply:
Used replace for all lines like this.